### ISSUE TITLE
Release 1.9.5 breaks ghost install

### ISSUE THREAD

**Issue by MaskyS** [TryGhost/Ghost-CLI#834]
### Summary
The script on DigitalOcean's one-click Ghost app no longer works, returning `ENOENT: no such file or directory, lstat '/home/ghost-mgr/.config'' ghost` Manually running `ghost install` from `/var/www/ghost` gave the same error.

### Steps to Reproduce

Note: Keep in mind I did this on a droplet that, when created, came with ghost-cli 1.9.4. I then rebuilt the droplet today, at which point I started getting the error. I'm not sure what effect this may have had, but mentioning it anyway.
1. Create a 1-click ghost app on DigitalOcean
2. SSH into the droplet, setup will start and install ghost-cli
3. Press enter when prompted to do so

Following error message: 

```
✖ Ensuring correct ~/.config folder ownership
An error occurred.
Message: 'ENOENT: no such file or directory, lstat '/home/ghost-mgr/.config''


Debug Information:
    OS: Ubuntu, v18.04
    Node Version: v8.10.0
    Ghost-CLI Version: 1.9.5
    Environment: production
    Command: 'ghost install --auto --db=mysql --dbhost=localhost --dbname=ghost_production --dbuser=ghost --dbpass=ce5d7f42ea7ac28d219d0def04a48e12c7fca6077ecfb944 --dir=/var/www/ghost --start'

```

### Log file

```
Debug Information:
    OS: Ubuntu, v18.04
    Node Version: v8.10.0
    Ghost-CLI Version: 1.9.5
    Environment: production
    Command: 'ghost install --auto --db=mysql --dbhost=localhost --dbname=ghost$
An error occurred.
Message: 'ENOENT: no such file or directory, lstat '/home/ghost-mgr/.config''

Stack: Error: ENOENT: no such file or directory, lstat '/home/ghost-mgr/.config'
Code: ENOENT
Path: /home/ghost-mgr/.config


```

## Bug submission checklist

Please fill out this checklist to acknowledge that you followed the requirements to submit a bug report.

- [x] Tried to find help in the forum & docs
- [x] Checked for existing issues
- [x] Attached log file
- [x] Provided technical details incl. operating system

## Resolving error
1. Install ghost-cli 1.9.4: `sudo npm i -g ghost-cli@1.9.4`
2. Follow through with [this guide](https://krisviceral.com/fixing-digitalocean-ghost-one-click-install-issues/) to obtain MySQL details, etc
3. Switch user using `sudo -i -u ghost-mgr`
4. Go to where you want to install ghost: `cd /var/www/ghost`
4. Run `ghost install` and enter details collected from Step 2

**Comment by acburdine** [comment]
@MaskyS confirmed, am working on a fix.

**Comment by acburdine** [comment]
@MaskyS Just released ghost-cli@1.9.6 which fixes the issue. Sorry for the trouble!

**Comment by MaskyS** [comment]
Awesome, thanks for the prompt fix :smiley:
